File: C/bitcount.c
#include <stdio.h>
#include <stdlib.h>
#include <errno.h>

static const int BITS = 64;

static int bitcount(int input) {
	int neg = input < 0 ? 1 : 0;
	input = neg ? (input * -1) - 1 : input;
	int count = 0;
	for (int i = 0; i < BITS; i++) {
		if (input & 1) {
			count++;
    }
		input = input >> 1;
	}
	return neg ? BITS - count : count;
}

int main(int argc, char* argv[]) {
	if (argc < 2) {
		printf("You must enter a number!\n");
		return 1;
	}
	int input = (int) strtol(argv[1], &argv[1], 10);
	if (errno == EINVAL) {
		printf("You must enter a number!\n");
		return 1;
	}
	printf("%d\n", bitcount(input));
	return 0;
}

File: MUMPS/bitcount.m
q
main()
 n input,done
 f  d  q:done
 . r "Please enter a number: ",input
 . i input?.N s done=1
 . e  w !,"You must enter a number!"
 w !,$$bitcount(input),!
 q
bitcount(input)
 n neg,counter,loop
 i 'input q 0
 i input<0 d
 . s neg=1
 . s input=(input*-1)-1
 f loop=1:1:32 d
 . i input#2 s counter=counter+1
 . s input=input\2
 i neg q 32-counter
 q counter
